Libraries/StyleSheet/normalizeColor.js

```diff
@@ -23,6 +27,21 @@ function normalizeColor(color: string | number): ?number {
     return null;
   }
 
+  if (typeof color === 'object' && color !== null) {
```

Comment:
This file has jest tests, can you add some new tests for this behavior at normalizeColor-test.js?